resources/build.sh

```diff
@@ -24,6 +24,7 @@ set -e
 : ${INPUT_PATH:="/input"}
 : ${CUSTOM_IMAGE_SCRIPT:="image.sh"}
 
+ALPINE_BRANCH=$(echo $ALPINE_BRANCH | sed '/^[^v].*[0-9]/s/^/v/')
```

Comment:
Nice idea. 

But I am not sure if I understand all cases covered by the regex.
I would say a simply "add a `v` if the version starts with a number" would be enough.

Something like:
```suggestion
ALPINE_BRANCH=$(echo $ALPINE_BRANCH | sed '/^[0-9]/s/^/v/')
```

Reply:
LGTM!